packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthroughs/auth-questions.js

```diff
@@ -440,11 +445,23 @@ function userPoolProviders(oAuthProviders, coreAnswers, prevAnswers) {
       }),
     );
     res.hostedUIProviderCreds = JSON.stringify(
-      oAuthProviders.map(el => ({
-        ProviderName: el,
-        client_id: coreAnswers[`${el.toLowerCase()}AppIdUserPool`],
-        client_secret: coreAnswers[`${el.toLowerCase()}AppSecretUserPool`],
-      })),
+      oAuthProviders.map(el => {
+        if (el === 'SignInWithApple') {
+          return {
+            ProviderName: el,
+            client_id: coreAnswers[`${el.toLowerCase()}ClientIdUserPool`],
```

Comment:
I'll stop pointing it out, but can you replace all of these `el.toLowerCase()` calls here and throughout the PR.